curiouslearning/FeedTheMonsterJS
# fix: CR-187 bangla json

# Changes
- /build
- lang/bangla/ftm_bangla.json

# How to test
- When I play levels 25, 26, and 27,
I should no longer see the dotted circle

Ref: [CR-187](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-187)


[CR-187]: https://curiouslearning.atlassian.net/browse/CR-187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
